[ERROR] No matching export in "node_modules/antd/es/index.js" for import "theme"

### Reproduction link

[https://stackblitz.com/edit/vitejs-vite-tmydrg?file=src%2FApp.tsx&terminal=dev](https://stackblitz.com/edit/vitejs-vite-tmydrg?file=src%2FApp.tsx&terminal=dev)

### Steps to reproduce

pro-components 官网 pro-layout 第一个示例

### What is expected?

组件正常渲染

### What is actually happening?

 [ERROR] No matching export in "node_modules/antd/es/index.js" for import "theme"

    node_modules/@ant-design/pro-provider/es/index.js:5:47:
      5 │ ...igProvider as AntdConfigProvider, theme as antdTheme } from 'antd';
        ╵                                      ~~~~~

✘ [ERROR] No matching export in "node_modules/antd/es/index.js" for import "theme"

    node_modules/@ant-design/pro-utils/es/useStyle/index.js:5:25:
      5 │ import { ConfigProvider, theme as antdTheme } from 'antd';
        ╵                          ~~~~~

Error: Build failed with 2 errors:
node_modules/@ant-design/pro-provider/es/index.js:5:47: ERROR: No matching export in "node_modules/antd/es/index.js" for import "theme"
node_modules/@ant-design/pro-utils/es/useStyle/index.js:5:25: ERROR: No matching export in "node_modules/antd/es/index.js" for import "theme"
    at failureErrorWithLog (/home/projects/vitejs-vite-tmydrg/node_modules/esbuild/lib/main.js:1618:15)
    at eval (/home/projects/vitejs-vite-tmydrg/node_modules/esbuild/lib/main.js:1260:28)
    at runOnEndCallbacks (/home/projects/vitejs-vite-tmydrg/node_modules/esbuild/lib/main.js:1040:63)
    at buildResponseToResult (/home/projects/vitejs-vite-tmydrg/node_modules/esbuild/lib/main.js:1258:7)
    at eval (/home/projects/vitejs-vite-tmydrg/node_modules/esbuild/lib/main.js:1371:14)
    at eval (/home/projects/vitejs-vite-tmydrg/node_modules/esbuild/lib/main.js:672:9)
    at handleIncomingPacket (/home/projects/vitejs-vite-tmydrg/node_modules/esbuild/lib/main.js:769:9)
    at Socket.readFromStdout (/home/projects/vitejs-vite-tmydrg/node_modules/esbuild/lib/main.js:638:7)
    at EventEmitter.emit (https://vitejs-vite-tmydrg.w.staticblitz.com/blitz.a1e315a41ec6498e2c2758b32fbf437db67b244b.js:6:155574)
    at addChunk (https://vitejs-vite-tmydrg.w.staticblitz.com/blitz.a1e315a41ec6498e2c2758b32fbf437db67b244b.js:6:656479) {
  errors: [
    {
      detail: undefined,
      id: '',
      location: [Object],
      notes: [],
      pluginName: '',
      text: 'No matching export in "node_modules/antd/es/index.js" for import "theme"'
    },
    {
      detail: undefined,
      id: '',
      location: [Object],
      notes: [],
      pluginName: '',
      text: 'No matching export in "node_modules/antd/es/index.js" for import "theme"'
    }
  ],
  warnings: []
}

| Environment | Info |
| --- | --- |
| antd | 4.23.0 |
| React | 18.2.0 |
| System | Windows 11 |
| Browser | Microsoft Edge 版本 105.0.1343.27 (正式版本) beta (64 位) |

---
构造工具使用的 VITE v3.1.0

<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/components/index.tsx
+++ b/components/index.tsx
@@ -16,6 +16,9 @@ if (
 }
 /* @remove-on-es-build-end */
 
+// Fix vite build error
+export const theme = null;
+
 export { default as Affix } from './affix';
 export type { AffixProps } from './affix';
 export { default as Alert } from './alert';

